### PR TITLE
Testing - add integration tests for enrichers sinking to Application Insights

### LINKED ISSUE #67: Provide integration tests for our enrichers
**Is your feature request related to a problem? Please describe.**
Provide integration tests for our enrichers so that we can verify them as well.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #67

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Does all of the info show up correctly?

**Comment by stijnmoreels** [comment]
> Does all of the info show up correctly?

Yes, I checked that.

File: src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs [hunk 1]
```diff
@@ -197,6 +193,92 @@ public void LogSqlDependency_SinksToApplicationInsights_ResultsInSqlDependencyTe
             }
         }
 
+        [Fact]
+        public void LogEventWithComponentName_SinksToApplicationInsights_ResultsInTelemetryWithComponentName()
+        {
+            // Arrange
+            string componentName = _bogusGenerator.Commerce.ProductName();
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithComponentName(componentName)))
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:201]: Can you add a test for component name with a dependency as well please?

This to verify that the application map shows a node with that name

File: src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs [hunk 2]
```diff
@@ -197,6 +193,92 @@ public void LogSqlDependency_SinksToApplicationInsights_ResultsInSqlDependencyTe
...
+            // Arrange
+            string operationId = $"operation-{Guid.NewGuid()}";
+            string transactionId = $"transaction-{Guid.NewGuid()}";
+            DefaultCorrelationInfoAccessor.Instance.SetCorrelationInfo(new CorrelationInfo(operationId, transactionId));
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithCorrelationInfo()))
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                // Act
+                logger.LogInformation("This message will be correlated");
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:241]: Let's track two messages and ensure that they are linked together.

Comment (stijnmoreels) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:241]: Added...
Is this what you want:
image

Comment (tomkerkhove) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs]: Yes, thanks!

File: src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs (hunk 1 above)
Comment (tomkerkhove) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:201]: Looks like this was added

Comment (stijnmoreels) [reply src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs:201]: Added...
Is this what you want:
image
image

**Pull request merged**